content/guide/07-element-directives.md

```diff
@@ -517,3 +517,96 @@ Use actions for things like:
 	}
 }
 ```
+
+#### Classes
+
+Classes let you toggle classes on and off on your elements.
```

Comment:
Perhaps add a line saying what the following example does? 'This example adds an `active` class to the appropriate `<li>` elements when etc. etc.'

Reply:
Also, please give an example of the syntax in use, outside of an example.